src/Domain/Creators/FrustalCloak.hpp

```diff
@@ -80,9 +83,29 @@ class FrustalCloak : public DomainCreator<3> {
     static constexpr Options::String help = {"The origin preimage in [x,y,z]."};
   };
 
-  using options = tmpl::list<InitialRefinement, InitialGridPoints,
-                             UseEquiangularMap, ProjectionFactor,
-                             LengthInnerCube, LengthOuterCube, OriginPreimage>;
+  template <typename BoundaryConditionsBase>
+  struct BoundaryCondition {
+    static std::string name() noexcept { return "BoundaryCondition"; }
+    static constexpr Options::String help =
+        "Options for the boundary conditions.";
```

Comment:
To me, that the way you described this for the Rectangle creator ("The boundary condition to impose on all sides.") was much clearer.